app/services/case_extend_sar_deadline_service.rb

```diff
@@ -0,0 +1,88 @@
+class CaseExtendSARDeadlineService
+  attr_reader :result, :error
+
+  def initialize(user, kase, extension_days, reason)
```

Comment:
I've copied the code from `CaseExtendForPITService` - is named parameters something we want to  make the default for method signatures?

Reply:
I think in all the tests there are loads of magic numbers, we could either put them in as named variables here so they make sense elsewhere or call them out as a different variable in the test.
I;m trying to avoid something like service.new(3) what is 3? I'd rather service.new(days_to_extend_by: 3) or days_to extend by = 3 service.new(days to extend by)

Reply:
Named variables?

Reply:
I've now changed the service to request named variables rather than only ensuring tests use descriptive variable names/named parameters.